### ISSUE TITLE
Add more real use cases with host app & engines 

### ISSUE THREAD

**Issue by villander** [ember-engines/ember-engines.com#62]
Include a few examples that lists a host app and engines, as well a few dependencies (common and shared) to illustrate points.

Lazy loading - https://github.com/ember-engines/ember-engines/issues/154#issuecomment-233160891

**Comment by KanagarajanK** [comment]
Yes Earlier as part of mounting engine with parent app, I have the doc for what are all the configs needs to be done in parent app/app.js under `engines` object now that info is missing

Edit: https://ember-engines.com/docs/links now this info is provided here, a little tricky to find at first, it would be great if a separate section which titled like how to mount with parent app
